Google.Api.Generator/Generation/SourceFileContext.cs

```diff
@@ -301,5 +301,50 @@ public ArrayTypeSyntax ArrayType(Typ arrayTyp, int? size = null)
         public virtual T Import<T>(System.Type t, T o) => o;
 
         public abstract CompilationUnitSyntax CreateCompilationUnit(NamespaceDeclarationSyntax ns);
+
+        protected CompilationUnitSyntax AddLicense(CompilationUnitSyntax cu)
+        {
+            // TODO: POssibly allow customization of license? Is this required?
+            // TODO: How should the copyright year be dealt with. would it be correct to always use the current year?
```

Comment:
That's what we do now, I believe - and it's probably a good enough starting point.

Reply:
OK, thanks.